presto-cli/src/main/java/io/prestosql/cli/FormatUtils.java

```diff
@@ -165,6 +165,24 @@ public static String formatTime(Duration duration)
         return format("%s:%02d", minutes, seconds);
     }
 
+    public static String formatFinalTime(Duration duration)
+    {
+        int totalSeconds = Ints.saturatedCast(duration.roundTo(SECONDS));
```

Comment:
I don't think we want to round here, since that would mean `59.5` would be the effective cutoff. We can do
```java
if (duration.toMillis() >= MINUTES.toMillis(1)) {